hw-1/task1.js

```diff
@@ -0,0 +1,19 @@
+import * as readline from 'readline';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on('line', (text) => {
+  console.log(`${text} -> ${reverse(text)}`);
```

Comment:
Minor: If the input is tabulation (TAB sign, but actually 6 spaces), then the output will be 2 spaces. Try to think, how to fix that. Example:
```
"       " -> "  "
```

Reply:
Interesting. Do you have any ideas how to fix it? I noticed it happens if I use string concatenation. Though, I changed output style and it works, but I'm still interested what causes it?

Reply:
You need to define `UTF-8` encoding for input

Reply:
Done! Thanks.